src/feature/Routes/index.tsx

```diff
@@ -2,10 +2,12 @@ import { NavigationContainer } from '@react-navigation/native';
 import { createStackNavigator } from '@react-navigation/stack';
 import React from 'react';
 import AuthStack, { AuthStackParamsList } from './AuthStack';
+import FloorStack, { FloorStackParamsList } from './FloorStack';
 import { Navigator } from '@/constants/screens';
 
 export type RootStackParamsList = {
   [Navigator.AuthStack]: AuthStackParamsList;
+  [Navigator.FloorStack]: FloorStackParamsList;
```

Comment:
🍌 이거 제가 처음에 잘못 작업해놔서 헷갈리셨을 것 같아요 ㅠㅠ 제 다른 PR에서는 고쳐놨는데, stack의 경우 `NavigatorScreenParams`를 사용해야 하더라구요!

```suggestion
  [Navigator.FloorStack]: NavigatorScreenParams<FloorStackParamsList>;
```